csharp/Test.cmd

```diff
@@ -5,19 +5,33 @@ SET CMDHOME=%~dp0
 @REM Remove trailing backslash \
 set CMDHOME=%CMDHOME:~0,-1%
 
-SET VSIDEDIR=%VS120COMNTOOLS%..\IDE
-SET VSTESTEXEDIR=%VSIDEDIR%\CommonExtensions\Microsoft\TestWindow
-SET VSTESTEXE=%VSTESTEXEDIR%\VSTest.console.exe
+if "%NUNITCONSOLE%" == "" (
+  GOTO :NUNITCONSOLEERROR
+)
 
 cd "%CMDHOME%"
 @cd
 
-SET CONFIGURATION=Release
+SET CONFIGURATION=Debug
```

Comment:
If only testing one configuration, isn't it more important to test Release build which most people will likely use?